project/ScalafixBuild.scala

```diff
@@ -211,6 +231,8 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     // don't publish scala 3 artifacts for now
     publish / skip := (if ((publish / skip).value) true
                        else scalaBinaryVersion.value == "3"),
+    publishLocal / skip := (if ((publishLocal / skip).value) true
+                            else scalaBinaryVersion.value == "3"),
```

Comment:
this turned out to have no effect as its intent is already achieved by the line just above, see https://github.com/scalacenter/scalafix/pull/1776